lib/guardian/plug/backdoor.ex

```diff
@@ -0,0 +1,76 @@
+defmodule Guardian.Plug.Backdoor do
+  @moduledoc """
+  This plug allows you to bypass authentication in acceptance tests by passing
+  the token needed to load the current resource directly to your Guardian module
+  via a query string parameter.
+
+  ## Installation
+
+  Add the following to your Phoenix router before other Guardian plugs.
+
+  ```
+  if Mix.env() == :test do
+    plug Guardian.Plug.Backdoor, module: MyApp.Guardian
+  end
+  plug Guardian.Plug.VerifySession
+  ```
+
+  NOTE: This plug is designed for acceptance testing and should never be added
+  to a production environment.
+
+  ## Usage
+
+  Now that `Guardian.Plug.Backdoor` is installed, it's time to sign in.
+
+  If you're using [Hound][hound], you can write the following code.
```

Comment:
Maybe we can remove the library specific examples from the moduledoc.  Instead, we can create a `Hound` section in the README.  @keathley can add his own sections later 👍 

Thoughts @jayjun?

Reply:
@doomspork You’re right, reference to other libraries belong in the README.